far/FarRus.hlf.m4

```diff
@@ -2608,15 +2610,15 @@ $ #Настройка панели#
 
 @TreeSettings
 $ #Настройки дерева папок#
-  #Автосмена папки#         Если эта опция включена, то передвижения
-                          курсора по ~дереву папок~@TreePanel@ будут вызывать
-                          смену папки в другой панели. Если эта опция
-                          выключена, то для смены папки из дерева
-                          папок вы должны нажать #Enter#.
+  #Автосмена папки#         Если эта опция включена, то передвижение курсора
+                          по ~дереву папок~@TreePanel@ будет синхронно менять папку в другой панели.
+                          Если эта опция выключена, то для смены папки из дерева папок
+                          вы должны нажать #Enter#.
+                          Соответствующий параметер ~far:config~@FarConfig@ -- #Panel.Tree.AutoChangeFolder#.
 
-  #Минимальное#             Минимальное количество папок на диске,
-  #количество папок#        после которого будет создаваться файл
-                          #tree3.far#.
+  #Минимальное#             Минимальное количество папок на диске, для которого
+  #количество папок#        будет создаваться файл кэша дерева каталогов #tree3.far#.
+                          Соответствующий параметер ~far:config~@FarConfig@ -- #Panel.Tree.MinTreeCount#.
```

Comment:
No need to specify far:config equivalent for every option,

Reply:
Removed.